src/workerd/api/node/process.h

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2023 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+#pragma once
+
+#include <workerd/api/node/i18n.h>
+#include <workerd/api/node/node-version.h>
+#include <workerd/jsg/jsg.h>
+#include <workerd/util/mimetype.h>
+
+namespace workerd::api::node {
+
+class ProcessModule final: public jsg::Object {
+ public:
+  ProcessModule() = default;
+  ProcessModule(jsg::Lock&, const jsg::Url&) {}
+
+  jsg::JsValue getBuiltinModule(jsg::Lock& js, kj::String specifier);
+
+// We will always set platform: 'linux' for edgeworker
```

Comment:
```suggestion
// We will always set platform: 'linux' for production
```

Reply:
If we write a test that runs on windows using process.platform, it will never run on EW? This is not we want, do we?

Reply:
Not sure what you're asking here @anonrig ... the internal CI builds only ever run on linux machines and the `EW_APPLICATION_NAME` flag is only ever defined on production builds. We want `process.platform` to always be `linux` internally. We can have the windows and mac values available on workerd since that's the only place it is relevant.

Reply:
Now I understand. Thanks folks!